twopaneview/src/TwoPaneView.tsx

```diff
@@ -0,0 +1,241 @@
+import React, {Fragment, Component} from 'react';
+import {
+  SafeAreaView,
+  StyleSheet,
+  ScrollView,
+  View,
+  ViewProps,
+  Text,
+  StatusBar,
+  Dimensions,
+  ViewStyle,
+} from 'react-native';
+import { DualScreenInfo } from 'react-native-dualscreeninfo'
+
+const TPV_Orientation = {
+  Horizontal: 'horizontal',
+  Vertical: 'vertical',
+};
+
+const TPV_PanePriority = {
+  Pane1: 'pane1',
+  Pane2: 'pane2',
+};
+
+const PaneMode = {
+  Auto: 'Auto',
+  Single: 'Single',
+  Dual: 'Dual',
+};
```

Comment:
This stuff isn't typescript.  The request about converting to typescript was really about being able to declare a type definition file so that consumers get "real" types, editors can use intell-sense, etc.  Feel free to follow-up on this later if necessary.

Reply:
Added types in types.ts